APP_DICT:
  description: Application for tracking a target based on AI detected objects
  pkg_name: nepi_app_ai_pt_tracker
  group_name: AUTOMATION
  config_file: app_ai_pt_tracker.yaml
  app_file: ai_pt_tracker_app_node.py
  node_name: app_ai_pt_tracker
RUI_DICT:
  rui_menu_name: PanTilt Object Tracker
  rui_files:
  - NepiAppPtTracker.js
  rui_main_file: NepiAppPtTracker.js
  rui_main_class: AiPtTrackerApp
